cypress-tests/cypress/e2e/PaymentTest/00024-ConnectorAgnosticNTID.cy.js

```diff
@@ -126,6 +152,33 @@ describe("Connector Agnostic Tests", () => {
       it("List Payment Method for Customer", () => {
         cy.listCustomerPMByClientSecret(globalState);
       });
+
+      it("Confirm No 3DS MIT", () => {
```

Comment:
Can you please confirm the payment with payment_token ,
You can modify the test case , or you can add additional test case